docs/en/api/wrapper/exists.md

```diff
@@ -1,20 +1,20 @@
 # `exists()`
 
-<p><strong>⚠Cette page est actuellement en cours de traduction française. Vous pouvez repasser plus tard ou <a href="https://github.com/vuejs-fr/vue-test-utils" target="_blank">participer à la traduction</a> de celle-ci dès maintenant !</strong></p><p>Assert `Wrapper` or `WrapperArray` exists.</p>
+Asserte que `Wrapper` ou `WrapperArray` existent.
 
-Returns false if called on an empty `Wrapper` or `WrapperArray`.
+Retourne `false` si appellé sur un `Wrapper` ou `WrapperArray` vide.
 
-- **Returns:** `{boolean}`
+- **Retourne :** `{boolean}`
 
-- **Example:**
+- **Exemple :**
 
 ```js
 import { mount } from '@vue/test-utils'
 import Foo from './Foo.vue'
 
 const wrapper = mount(Foo)
 expect(wrapper.exists()).toBe(true)
-expect(wrapper.find('does-not-exist').exists()).toBe(false)
+expect(wrapper.find('existe-pas').exists()).toBe(false)
 expect(wrapper.findAll('div').exists()).toBe(true)
-expect(wrapper.findAll('does-not-exist').exists()).toBe(false)
+expect(wrapper.findAll('existe-pas').exists()).toBe(false)
```

Comment:
Attention, j'ai un doute sur cette traduction : j'ai peur que l'assertion ne fonctionne plus si on traduit 'does-not-exist'